### PR TITLE
Forum: Add global `default_view` option (ILIAS 9 Feature)

### PR DESCRIPTION
**Pull request by lscharmer**
This PR applies feature request: https://docu.ilias.de/goto_docu_wiki_wpage_7720_1357.html.

### CODE REVIEW

File: Modules/Forum/classes/class.ilObjForumAdministrationGUI.php [hunk 1]
```diff
@@ -92,143 +100,113 @@ public function getAdminTabs(): void
...
-        $form->addItem($check);
+        $items = [];
 
-        $file_upload = new ilRadioGroupInputGUI(
-            $this->lng->txt('file_upload_allowed_fora'),
-            'file_upload_allowed_fora'
+        $checkbox = fn ($label) => $field->checkbox($this->lng->txt($label), $this->lng->txt($label . '_desc'));
+        $nice = fn ($label) => sprintf('%s (%s)', $this->lng->txt('sort_by_date'), $this->lng->txt($label));
+        $to_string = static fn ($x) => (string) $x;
+        $withOptions = static fn ($x, $options) => array_reduce(
```
Comment (mjansenDatabay) [inline Modules/Forum/classes/class.ilObjForumAdministrationGUI.php]: New variable names should be `snake_case`.

Comment (lscharmer) [reply Modules/Forum/classes/class.ilObjForumAdministrationGUI.php]: Done

**Comment by mjansenDatabay** [comment]
@lscharmer Could you please resolve the conflict? Thx!

**Comment by lscharmer** [comment]
@mjansenDatabay the conflicts are resolved.

**Pull request merged**